LICENSE

```diff
@@ -1,15 +1,15 @@
   Copyright (c) 2009-2017 Dave Gamble and cJSON contributors
- 
+
```

Comment:
Oh, I didn't know there were whitespaces, even though I have a plugin that highlights those in red. hm.

Better just remove all the spaces at the beginning of the line in this LICENSE file, please do that.